.storybook/preview.js

```diff
@@ -59,4 +60,21 @@ if (!isChromatic()) {
   }));
 }
 addDecorator(withA11y);
-addDecorator(withThemeSelector);
\ No newline at end of file
+addDecorator(withThemeSelector);
+
+addDecorator((Story, { id }) => {
+  const callback = (id, phase, actualTime, baseTime, startTime, commitTime) => {
+    console.log(`Starting gathering performance data for ${id} component`);
+    console.log(`${id}'s ${phase} phase:`);
+    console.log(`Actual time: ${actualTime}`);
+    console.log(`Base time: ${baseTime}`);
+    console.log(`Start time: ${startTime}`);
+    console.log(`Commit time: ${commitTime}`);
+    console.log(`End of gathering performance data for ${id} component`);
+  };
+  return (
+    <Profiler id={id} onRender={callback}>
+      <Story />
+    </Profiler>
+  );
+});
```

Comment:
This code in `preview.js` won't work after storybook is updated to v6

Reply:
New API https://storybook.js.org/docs/react/writing-stories/decorators#global-decorators